drivers/SmartThings/matter-switch/fingerprints.yml

```
@@ -71,6 +71,17 @@ matterManufacturer:
     vendorId: 0x1312
     productId: 0x01
     deviceProfileName: light-level-colorTemperature-2710k-6500k
+#Zemismart
+  - id: "5020/43524"
+    deviceLabel: "Zemismart Thread Smart Switch"
+    vendorId: 0x139C
+    productId: 0xAA04
+    deviceProfileName: "switch-binary"  
```

```suggestion
    deviceProfileName: switch-binary 
```
nit: just formatting - all other fingerprints don't have quotes around the `deviceProfileName`